packages/dendron-next-server/pages/_app.tsx

```diff
@@ -79,6 +79,8 @@ function AppVSCode({ Component, pageProps }: any) {
     logger.info("AppVSCode.init");
   }, []);
 
+  // query only has ws and port properties. To intitalize engine with useEngine, we require url as well.
+  query.url = `http://localhost:${query.port}`;
```

Comment:
Could you use `getWorkspaceParamsFromQueryString` here instead? It's used just below this as well.

Reply:
I did try this out but was getting `window is not defined` error. Let me give that a another try.